spec/features/invite_user_spec.rb

```diff
@@ -48,6 +48,9 @@ def register_with_valid_card(invitation={})
     select(valid_card[:exp_month], :from => 'exp_month' )
     select(valid_card[:exp_year], :from => 'exp_year' )
     click_button 'Sign Up'
+
+    sleep 3.0 # wait for the filling things done avoid random fails with capybara-webkit
```

Comment:
You can also adjust the default wait time for Capybara, if you bump it up to 5 seconds from the default 2, it will have the same effect and will be app-wide.